backend/src/schema/resolvers/users.spec.js

```diff
@@ -542,6 +542,12 @@ describe('DeleteUser', () => {
               mutate({ mutation: deleteUserMutation, variables }),
             ).resolves.toMatchObject(expectedResponse)
           })
+
+          it('deletes user avatar and post hero images', async () => {
+            await expect(neode.all('Image')).resolves.toHaveLength(22)
+            await mutate({ mutation: deleteUserMutation, variables })
+            await expect(neode.all('Image')).resolves.toHaveLength(20)
+          })
```

Comment:
where are these magic numbers coming from?

Reply:
Well, I don't know myself, I just ran the test and checked how many images are set up in the `beforeEach`s prior to the test case :point_up: then I wanted it to remove more than 1 image.

Reply:
humm... don't really like that